psydac/feec/derivatives.py

```diff
@@ -356,6 +361,16 @@ def copy(self):
 
 #====================================================================================================
 class DiffOperator:
```

Comment:
Btw, why is DiffOperator not subclassing LinearOperator ?

Reply:
`DiffOperator` was implemented before Julian's work on `LinearOperator`s. There are things that we would need to fix in order to subclass it, for instance, now `DiffOperator` works with `FemField`s and not `StencilVector`s. However this would be another PR.

Reply:
ok thanks !